lib/modules/onc_program/onc_smart_discovery_sequence.rb

```diff
@@ -267,7 +287,7 @@ def after_save_oauth_endpoints(oauth_token_endpoint, oauth_authorize_endpoint)
           description %(
             A SMART on FHIR server SHALL convey its capabilities to app
             developers by listing a set of the capabilities. The following
-            capabilities are required: #{REQUIRED_SMART_CAPABILITIES.join(', ')}
+            capabilities are required: #{OncSMARTDiscoverySequence.required_smart_capabilities.join(', ')}
```

Comment:
This shows the same list of required capabilities for both standalone and ehr launches:

Standalone:
![Screen Shot 2021-02-18 at 8 34 38 AM](https://user-images.githubusercontent.com/15969967/108364652-50ea6780-71c4-11eb-940b-ce269a66a674.png)

EHR:
![Screen Shot 2021-02-18 at 8 35 16 AM](https://user-images.githubusercontent.com/15969967/108364668-55af1b80-71c4-11eb-8a4f-1b3809582243.png)

Reply:
Ah, good catch.  Unfortunately I'm not sure if there is any great way of fixing this without changing the code sharing strategy here, since this is executed within the context of a Test and doesn't know which sequence it is a part of.  I think the best fix here is for us to just describe both here.  e.g. "For patient launches, we are checking these capabilities ..., for ehr practitioner launches we are testing these other ones ...".  Thougths?

Reply:
I think that is an appropriate solution. Otherwise it would be a lot of work just to change the description.

Reply:
It turns out this is quite a pain to list the capabilities, because the 'parent' sequence doesn't have access to the 'child' sequence's required capabilities for the description, and requiring us to load that file caused some wierd behavior.  So I just left the language somewhat vague as to exactly which are required in each particular scenario in the description itself.  I think it should be self-evident based on errors, if they come up.  Does that sound reasonable?

Reply:
Yeah, sure.